app/helpers/compute_resources_helper.rb

```diff
@@ -43,8 +43,10 @@ def memory_options(max_memory)
     opts.map{|n| [number_to_human_size(n*Foreman::SIZE[:giga]), (n*Foreman::SIZE[:giga]).to_i] unless n > (max_memory / Foreman::SIZE[:giga])}.compact
   end
 
-  def password_placeholder(obj)
-    obj.id ? "********" : ""
+  def password_placeholder(obj, attr = nil)
+    pass ||=  !obj.read_attribute(attr).empty? || obj.has_attribute?(:password_hash)
```

Comment:
a) as  @alongoldboim commented inline, "I think you can take the ||= off and just place =, pass can't have a value from before."
b) extra space after `||=`
c) use .present? instead of !..empty?
d) what's password_hash for, is that a workaround for the User model?